Apps/Inc/ReadTritium.h

```diff
@@ -15,29 +14,48 @@
 #include "Tasks.h"
 
 /**
- * Motor Error States
- * Read messages from motor in ReadTritium and trigger appropriate error messages as needed based on bits
- * 
- */
+ * @enum tritium_error_code_t
+ * @brief Motor Error States
+ * Read messages from motor in ReadTritium and trigger appropriate 
+ * error messages as needed based on bits
+*/
 typedef enum{
+    /** hardware over current error */
     T_HARDWARE_OVER_CURRENT_ERR = (1<<0), 
-    T_SOFTWARE_OVER_CURRENT_ERR = (1<<1), 
-    T_DC_BUS_OVERVOLT_ERR = (1<<2), 
+    /** software over current error */
```

Comment:
This is minor, but the rest of the files have comments starting with a capital letter. These comments should probably be updated to reflect that.